### PR TITLE
QD-638 add cli args

### PR DESCRIPTION
**Pull request by sepich**
https://youtrack.jetbrains.com/issue/QD-638

### CODE REVIEW

**Review by jk — APPROVED** [review]

**Comment by jk** [comment]
> https://youtrack.jetbrains.com/issue/QD-638

Thanks for the timely consideration and code change. Looks good to me.

File: Docker/README.md [hunk 1]
```diff
@@ -48,64 +48,82 @@ To run analysis __locally__:
 
    where `source-directory` and `output-directory` are full local paths to, respectively, the project source code directory and the analysis results directory.
 
-   This command will run the analysis on your source code and start the web server to provide a convenient view of the results. Open [`http://localhost:8080`](http://localhost:8080) in your browser to examine the found problems and performed checks. Here, you can also reconfigure the analysis. See the [UI section](../UI/README.md) of this guide for details.
+   This command will run the analysis on your source code and start the web server to provide a convenient view of the results. Open [`http://localhost:8080`](http://localhost:8080) in your browser to examine the found problems and performed checks. Here, you can also reconfigure the analysis. See the [UI section](../UI/README.md) of this guide for details. Press `Ctrl-C` in docker console when you are done, to stop the web server.
```
Comment (zeitgen) [inline Docker/README.md:51]: Let's mb go the other way around? "When done, you can stop the web server by pressing `Ctrl-C` in the Docker console."

File: Docker/README.md [hunk 2]
```diff
@@ -48,64 +48,82 @@ To run analysis __locally__:
...
     docker run --rm \
         -v <source-directory>/:/data/project/ \
         -v <output-directory>/:/data/results/ \
         jetbrains/qodana
    ```
-
    where `source-directory` and `output-directory` are full paths to, respectively, the project source code directory and the analysis results directory. 
-   The output of `output-directory` is described [here](../General/output.md#basic-output).
+   The output of `output-directory` is described [here](../General/output.md#basic-output).  
+   Consider using [fail-thresold](https://github.com/JetBrains/Qodana/blob/main/General/qodana-yaml.md#fail-threshold) to make build fail on certain number of problems reached .
```
Comment (zeitgen) [inline Docker/README.md:76]: > reached .

There's an unneeded space before the period.

File: Docker/techs.md [hunk 3]
```diff
@@ -16,12 +40,12 @@ Examples of execution tuneup:
...
    ```
 
-- Save a report in HTML. The HTML report will be stored in a separate `report/` subdirectory under the `results` directory.
+- Save a report in HTML. By default, the HTML report will be stored in a separate `report/` subdirectory under the `results` directory. This location could be configured with `--report-dir`.
    ```
    docker run ... <image-name> --save-report
    ```
 
-- Display a report in HTML. After the inspection is finished, the container will not exit and will listen to port `8080`. You can connect to [`http://localhost:8080`](http://localhost:8080) to see the results.
+- Display a report in HTML. After the inspection is finished, the container will not exit and will listen to port `8080`. You can connect to [`http://localhost:8080`](http://localhost:8080) to see the results. Press Ctrl-C when done.
```
Comment (zeitgen) [inline Docker/techs.md:48]: I suggest reusing the above: "When done, you can stop the web server by pressing Ctrl-C in the Docker console."

**Review by zeitgen — APPROVED** [review]
Pls see a couple of suggestions below. Otherwise lgtm!

**Review by jk — APPROVED** [review]

**Review by ak1113 — APPROVED** [review]

**Pull request merged**